api/job.proto

```diff
@@ -0,0 +1,94 @@
+// Copyright 2018 The OpenPitrix Authors. All rights reserved.
+// Use of this source code is governed by a Apache license
+// that can be found in the LICENSE file.
+
+syntax = "proto3";
+package openpitrix;
+
+// set go package name to pb
+option go_package = "pb";
+
+import "google/protobuf/wrappers.proto";
+import "google/protobuf/timestamp.proto";
+import "google/api/annotations.proto";
+import "protoc-gen-swagger/options/annotations.proto";
+
+option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
+	info: {
+		title: "OpenPireix Project";
+		version: "0.0.1";
+		contact: {
+			name: "OpenPireix Project";
+			url: "https://openpitrix.io";
+		};
+	};
+	schemes: HTTP;
+	consumes: "application/json";
+	produces: "application/json";
+};
+
+
+message CreateJobRequest {
+	string _ = 1; // placeholder for job_id
```

Comment:
`_` is Go syntax, not proto file.

do u mean keep the `1` reserve?

Reply:
Yes, keep the 1 reserve, now job_id is useless because it is generate when handle the req.